Framework/DataHandling/src/SaveNXcanSAS.cpp

```diff
@@ -722,26 +760,36 @@ void SaveNXcanSAS::init() {
   declareProperty(std::make_unique<Mantid::API::FileProperty>("Filename", "", API::FileProperty::Save, ".h5"),
                   "The name of the .h5 file to save");
 
-  std::vector<std::string> radiation_source{"Spallation Neutron Source",
-                                            "Pulsed Reactor Neutron Source",
-                                            "Reactor Neutron Source",
-                                            "Synchrotron X-ray Source",
-                                            "Pulsed Muon Source",
-                                            "Rotating Anode X-ray",
-                                            "Fixed Tube X-ray",
-                                            "neutron",
-                                            "x-ray",
-                                            "muon",
-                                            "electron"};
+  std::vector<std::string> radiationSourceOptions{"Spallation Neutron Source",
+                                                  "Pulsed Reactor Neutron Source",
+                                                  "Reactor Neutron Source",
+                                                  "Synchrotron X-ray Source",
+                                                  "Pulsed Muon Source",
+                                                  "Rotating Anode X-ray",
+                                                  "Fixed Tube X-ray",
+                                                  "neutron",
+                                                  "x-ray",
+                                                  "muon",
+                                                  "electron"};
   declareProperty("RadiationSource", "Spallation Neutron Source",
-                  std::make_shared<Kernel::StringListValidator>(radiation_source), "The type of radiation used.");
+                  std::make_shared<Kernel::StringListValidator>(radiationSourceOptions), "The type of radiation used.");
+  std::vector<std::string> const geometryOptions{Sample::CYLINDER, Sample::FLAT_PLATE, Sample::DISC};
+  declareProperty("Geometry", std::string(Sample::DISC), std::make_shared<Kernel::StringListValidator>(geometryOptions),
```

Comment:
I've been thinking about this new `Geometry` property and backwards compatibility. If someone had used `SaveNXcanSAS` in a script that they had written using a previous version of Mantid and then ran that script in a version of Mantid that contains these changes, their saved file will contain a geometry entry set to the default of `Disc` where previously it wouldn't have contained any information. Similarly I guess it might also affect re-running history depending on what someone has done with a workspace. I'm wondering if we should consider making this parameter optional/setting the default to an empty string and only saving out geometry information if it's been passed in?

Reply:
I've asked Steve about this. He's suggested that we have some other value such as `Unknown` as the value should be set on the file by IBEX when making the run. I'm not sure how this fits in with our sample objects.